docs/modules/ROOT/pages/free-trial.adoc

```diff
@@ -31,4 +33,11 @@ You'll also receive emails from Hazelcast with useful resources to help you get
 At the end of the 14-day trial period, two things happen:
 
 - Your account is automatically suspended. This means that although you can still access the trial environment, your cluster is stopped and cannot be restarted.
-- Your trial environment, including all data, will be automatically deleted.
+- Your trial environment, including all data, is automatically deleted.
+
+Hazelcast will be in touch during your trial, and when it ends. If you want to talk further with us about subscription options, mailto:viridian@hazelcast.com[send us an email], or select the button available at the end of most trial communications from Hazelcast.
```

Comment:
Yes, of course I won't be merging any changes until they are ready; I just want the changes to be ready to go as I'm sure it will all happen very quickly when it happens.